### PR TITLE
qt5{,-static}: add pcre2 as dependency (fixes #3036)

### LINKED ISSUE #3036: mingw-w64-x86_64-qt5-5.9.2-1 broken
After upgrading to the last version, all .exe of the packages don't work, they ask for .dll, like Qt5Core.dll
Doing the downgrade resolves the problem.
I have attached both ldd output of Qt5Core.dll

[qt5-5.9.2-1.txt](https://github.com/Alexpux/MINGW-packages/files/1377580/qt5-5.9.2-1.txt)
[qt5-5.9.1-1.txt](https://github.com/Alexpux/MINGW-packages/files/1377587/qt5-5.9.1-1.txt)

### PR DESCRIPTION
**Pull request by Optiligence**

### CODE REVIEW

File: mingw-w64-qt5-static/PKGBUILD [hunk 1]
```diff
@@ -124,6 +124,7 @@ depends=("${MINGW_PACKAGE_PREFIX}-gcc-libs"
             "${MINGW_PACKAGE_PREFIX}-libwebp" \
             $([[ "$_with_openssl" == "yes" ]] && echo "${MINGW_PACKAGE_PREFIX}-openssl") \
             $([[ "$_system_pcre" == "yes" ]] && echo "${MINGW_PACKAGE_PREFIX}-pcre") \
+            $([[ "$_system_pcre" == "yes" ]] && echo "${MINGW_PACKAGE_PREFIX}-pcre2") \
```
Comment (Alexpux) [inline mingw-w64-qt5-static/PKGBUILD]: Well I thin you need not add another pcre but replace pcre with pcre2

File: mingw-w64-qt5/PKGBUILD [hunk 2]
```diff
@@ -124,6 +124,7 @@ depends=("${MINGW_PACKAGE_PREFIX}-gcc-libs"
             "${MINGW_PACKAGE_PREFIX}-libwebp" \
             $([[ "$_with_openssl" == "yes" ]] && echo "${MINGW_PACKAGE_PREFIX}-openssl") \
             $([[ "$_system_pcre" == "yes" ]] && echo "${MINGW_PACKAGE_PREFIX}-pcre") \
+            $([[ "$_system_pcre" == "yes" ]] && echo "${MINGW_PACKAGE_PREFIX}-pcre2") \
```
Comment (Alexpux) [inline mingw-w64-qt5/PKGBUILD]: The same

**Comment by Optiligence** [comment]
I uninstalled pcre and started qt assistant and that didn’t work so i left it in there. 
it’s qt5gui → harfbuzz → glib → pcre
but i guess glib has its own pcre dep, i will change it

**Comment by Optiligence** [comment]
i also bumped qt-creator now because its post_install script failed when pcre2 wasn’t installed

**Comment by Alexpux** [comment]
I will accept fixes but I will not rebuild qt5 in near time as it take too much time

**Pull request merged**